webhook/forward.go

```diff
@@ -99,9 +101,9 @@ type wsEventReceived struct {
 	Body   []byte
 }
 
-func runFwd(out io.Writer, port int, token, wsURL string, activateHook func() error) error {
+func runFwd(out io.Writer, port int, path string, token, wsURL string, activateHook func() error) error {
```

Comment:
Nit: No need to specify type twice if all consecutive arguments share the same type. 

```suggestion
func runFwd(out io.Writer, port int, path, token, wsURL string, activateHook func() error) error {
```

Reply:
Nice! I didn't know that - I have literally zero Go experience.